### PR TITLE
lunasvg: add version 3.0.0

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **lunasvg/3.0.0**

#### Motivation
This is a major version up.

#### Details
https://github.com/sammycage/lunasvg/compare/v2.4.1...v3.0.0

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Review by ErniGH — COMMENTED** [review]
disable "LUNASVG_BUILD_EXAMPLES"

**Review by ErniGH — APPROVED** [review]
Thanks

File: recipes/lunasvg/all/test_v1_package/CMakeLists.txt [hunk 1]
```diff
@@ -8,4 +8,14 @@ find_package(lunasvg REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE lunasvg::lunasvg)
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
+if (lunasvg_VERSION VERSION_LESS_EQUAL "2.3.2")
```
Comment (AbrilRBS) [inline recipes/lunasvg/all/test_v1_package/CMakeLists.txt:11]: I think this cxx version logic is wrong - 3.0.0 should have c++17 right?

Comment (toge) [reply recipes/lunasvg/all/test_v1_package/CMakeLists.txt:11]: @AbrilRBS 
Thank you for your review!
I fixed these conditions.

**Review by AbrilRBS — APPROVED** [review]
Thanks!

**Review by ErniGH — APPROVED** [review]

**Comment by AbrilRBS** [comment]
The CLA got bot confused, fixed now, sorry for the delay!

**Pull request merged**